cms/djangoapps/contentstore/views/item.py

```diff
@@ -1036,7 +1038,12 @@ def _get_release_date(xblock, user=None):
     Returns the release date for the xblock, or None if the release date has never been set.
     """
     # If year of start date is less than 1900 then reset the start date to DEFAULT_START_DATE
-    if xblock.start.year < 1900 and user:
+    try:
+        xblock_start_year = xblock.start.year
+    except ValueError:
+        xblock_start_year = 1899
```

Comment:
I see what you're doing here, but it feels hacky to me. Can you set xblock_start_year to `None` instead, and check for it being `None` on L1046?